### PR TITLE
Minor editorial comments

### PR DESCRIPTION
**Pull request by prince-chrismc**

### CODE REVIEW

File: docs/1.0. Overview.md [hunk 1]
```diff
@@ -20,4 +20,4 @@ There is some [prerequisite knowledge](2.0.%20Prerequisites.md) needed and some
 The use of OAuth 2.0 terminology has been adopted in the IS-10 standard. OAuth 2.0 was originally developed for allowing web apps to access protected resources, where the web app is the Client and the protected resource is the Resource Server. However, the NMOS Node acts as both a Client (when, for instance, registering with the NMOS Registry) AND as a Resource Server (when, for instance, accepting a connection request). For this reason the Node is sometimes referred to as the Client and sometimes as the Resource Server depending on context.
 
 ## Specification Precedence
-This guide is not a substitute for the NMOS Specifications. If there are any inconsistencies between this guide and the specifications, please assume that it is the specification that is correct.
+This guide is not a substitute for the NMOS Specifications. If there are any inconsistencies between this guide and the specifications, unequivocally the specification is correct.
```
Comment (prince-chrismc) [inline docs/1.0. Overview.md:23]: I felt the language should be more affirmative here

Comment (garethsb) [reply docs/1.0. Overview.md:23]: Agree. (@jonathanthorpe-sony is moving this section and adding some other words I suggested.)

File: docs/3.2. NMOS Private Claims.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # Private Claims
 _(c) AMWA 2021, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
 
-IS-10 has defined some private claims, related to the NMOS scopes. These claim names begin `x-nmos-` followed by the namespace identifier found in the URL for that given API (e.g. 'registration', 'query', 'connection', etc.). Each NMOS private claim specifies the read and write permission. Wildcards ('*') can be used to fine tune the level of access. In the following example, the token has been given full read and write permission for accessing the`registry.example.com` IS-04 Registration API.
+[IS-10 has defined some private claims](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.4._Behaviour_-_Access_Tokens.html#private-claims), related to NMOS systems. These claim names begin `x-nmos-` followed by the namespace identifier found in the URL for that given API (e.g. 'registration', 'query', 'connection', etc.). Each NMOS private claim specifies the read and write permission for that API. Wildcards ('\*') can be used to fine tune the level of access.
```
Comment (prince-chrismc) [inline docs/3.2. NMOS Private Claims.md:4]: I think that "scopes" may be confusing with that of the access token in the example below. So I am proposing this

```diff
-related to the NMOS scopes
+related to NMOS systems
```

Along with a link and fixing some formatting 😄

File: docs/4.6. Validating Access Tokens.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ _(c) AMWA 2021, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
...
+- Verify the Signature, by generating a new Base64url-encoded Signature using the issuer's public key and verify that it matches the original Signature, to ensure the token has not been tampered with. In event the issuer [public key](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys) is not known, it should attempt to obtain the missing public key via the the token `iss` claim as specified in [RFC 8414](https://tools.ietf.org/html/rfc8414 "OAuth 2.0 Authorization Server Metadata") section 3, and temporarily respond with an HTTP 503 code in order to avoid blocking the incoming authorized request.
```
Comment (prince-chrismc) [inline docs/4.6. Validating Access Tokens.md:7]: > In event the issuer [public key](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys) is not known, it should attempt to obtain the missing public key via the the token `iss` claim as specified in [RFC 8414]

I believe this section is assuming that the JWKs is cached by the client but I am a little unsure why it's "not known"

File: docs/1.0. Overview.md (hunk 1 above)
Comment (jonathan-r-thorpe) [reply docs/1.0. Overview.md:23]: I shall ensure the new section's wording is suitably emphatic.

File: docs/3.2. NMOS Private Claims.md (hunk 2 above)
Comment (jonathan-r-thorpe) [reply docs/3.2. NMOS Private Claims.md:4]: Agreed.  This section no longer exists in this doc, but I will try to remove reference to scopes where it is confusing.

File: docs/4.6. Validating Access Tokens.md (hunk 3 above)
Comment (jonathan-r-thorpe) [reply docs/4.6. Validating Access Tokens.md:7]: One for @simonlo-sony to answer I think

File: docs/1.0. Overview.md (hunk 1 above)
Comment (jonathan-r-thorpe) [reply docs/1.0. Overview.md:23]: Added to #31

File: docs/3.2. NMOS Private Claims.md (hunk 2 above)
Comment (jonathan-r-thorpe) [reply docs/3.2. NMOS Private Claims.md:4]: This was covered by #32

**Pull request merged**